packages/core/src/components/overlay/overlay.tsx

```diff
@@ -359,13 +359,11 @@ export class Overlay extends AbstractPureComponent2<OverlayProps, IOverlayState>
         }
 
         // add a special class to each child element that will automatically set the appropriate
-        // CSS position mode under the hood. also, make the container focusable so we can
-        // trap focus inside it (via `enforceFocus`).
+        // CSS position mode under the hood.
         const decoratedChild =
             typeof child === "object" ? (
                 React.cloneElement(child as React.ReactElement, {
                     className: classNames((child as React.ReactElement).props.className, Classes.OVERLAY_CONTENT),
-                    tabIndex: this.props.enforceFocus || this.props.autoFocus ? 0 : undefined,
```

Comment:
removing this fixes an issue w/ the omnibar where the container _and_ the input are both focusable, even though you probably don't want the container to be focusable in that situation. now that we have focus traps, this doesn't seem necessary. if consumers want non-focusable elements to be focusable, they will need to take care to set tabindex themselves. not sure if this constitutes a break